test/kuttl/multi-dcs/01-assert.yaml

```diff
@@ -1,3 +1,10 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 500
+commands:
+  - script: "kubectl get pods -n $NAMESPACE -o json | jq '.items | length' | jq 'if . != 5 then halt_error(-2) else \"OK\" end' 2> /dev/null"
```

Comment:
You can chain calls in jq:
```
$ pbpaste|jq 'if(.topping|length) > 5 then "ok" else "nok" end' 
"ok"
```

Reply:
Oh ok I had a problem with '| length' that was interpreting the rest of the line but with "( )" it should work. I pushing a new version right away

Reply:
Seems like the problem its still there, I will keep the previous one :/

Reply:
what problem ? I've tested and it works as expected